src/System.Security.Cryptography.Algorithms/tests/PKCS1MaskGenerationMethodTest.cs

```diff
@@ -72,7 +72,7 @@ public static void NegativeReturnParameterTest()
         public static void GenerateMaskTest()
```

Comment:
It may be worthwhile to make this test work for non-sha1 so we have more coverage for those scenarios since they're now supported. I'm not certain where to get these values if we do that, though.